'make deb' and 'make rpm' are broken

to fix #59 they were moved. I can't fix them right now, but it would be great if someone did.

Hi ,

I'm still seeing the same issue while installing rtslib package.Is there any workaround or patch to resolve the error in building the rpm package? 

# make rpm
Exporting the repository files...
Cleaning up the target tree...
Fixing version string...
Generating rpm specfile from template...
Generating rpm changelog...
Generating debian changelog...
Generated release tarball:
    dist/rtslib-fb-2.1.fb59.12.g845606a.tar.gz
Building rpm packages...
error: File /root/new/rtslib-fb/dist/rtslib-2.1.fb59.12.g845606a.tar.gz: No such file or directory
make: *** [build/rpm-stamp] Error 1
# 

Thanks,
Ravindar

Thanks Andy for your quick fix. We also see the similar error in compiling "configshell-fb".

 make rpm
Exporting the repository files...
Cleaning up the target tree...
Fixing version string...
Generating rpm specfile from template...
Generating rpm changelog...
Generating debian changelog...
Generated release tarball:
    dist/configshell-fb-1.1.fb19.1.g773dcad.tar.gz
Building rpm packages...
error: File /root/new/configshell-fb/dist/configshell-1.1.fb19.1.g773dcad.tar.gz: No such file or directory
make: *** [build/rpm-stamp] Error 1
[root@Ravindar configshell-fb]#

--- a/Makefile
+++ b/Makefile
@@ -66,6 +66,7 @@ build/release-stamp:
 			mv $${spectmpl} $$(basename $${spectmpl} .tmpl); \
 		done; \
 		rm -r example-rpm
+	@mv build/${PKGNAME}-${VERSION}/example-debian build/${PKGNAME}-${VERSION}/debian
 	@echo "Generating rpm changelog..."
 	@( \
 		version=$$(basename $$(git describe HEAD --tags | tr - .)); \
@@ -93,7 +94,7 @@ build/release-stamp:
 		echo; \
 		echo " -- $${author}  $${date}"; \
 		echo; \
-	) > build/${PKGNAME}-${VERSION}/example-debian/changelog
+	) > build/${PKGNAME}-${VERSION}/debian/changelog
 	@find build/${PKGNAME}-${VERSION}/ -exec \
 		touch -t $$(date -d @$$(git show -s --format="format:%at") \
 			+"%Y%m%d%H%M.%S") {} \;

